iac/main/resources/sustainability.yml

```diff
@@ -0,0 +1,44 @@
+SustainabilityBucket:
+  Type: 'AWS::S3::Bucket'
+  Properties:
+    AccessControl: Private
+    BucketName: !Sub ${Environment}-dap-sustainability
+    LoggingConfiguration:
+      DestinationBucketName: !Ref GlobalLogBucket
+      LogFilePrefix: dap-sustainability/log
+    PublicAccessBlockConfiguration:
+      BlockPublicAcls: true
+      BlockPublicPolicy: true
+      IgnorePublicAcls: true
+      RestrictPublicBuckets: true
+    VersioningConfiguration:
+      Status: Enabled
+    LifecycleConfiguration:
```

Comment:
i think because of object versioning, the files dont actually get deleted. this policy add the delete marker but doesnt actually delete the object. 

https://repost.aws/knowledge-center/s3-empty-bucket-lifecycle-rule and the rules we need are in our other templates